err_size.go

```diff
@@ -0,0 +1 @@
+package quic
```

Comment:
We probably don't need this file?

Reply:
I think the `// +build unix` flag is not yet merged on Go upstream yet. I need to make some changes.

Reply:
ok, i moved _generic into this file instead.